docs/unique/board-automation-improvements.md

```
@@ -476,22 +476,22 @@
 Automation of the board needs some serious work. A lot of these automaticly generated tasks were sort of hidden from me til I fixed the scripts enough for them to not totally ruin the board.
 on Update
   - run-step-api
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)